README.md

```diff
@@ -11,6 +11,7 @@ Rust-Bio-Tools provides a command `rbt`, which currently supports the following
 * a linear time extraction of depth information from BAMs at given loci (`rbt bam-depth`)
 * a utility to quickly filter records from a FASTQ file (`rbt fastq-filter`)
 * a tool to merge FASTQ reads using unique molecular identifiers (UMIs) (`rbt call-consensus-reads`)
+* a tool to merge BAM reads based on duplicates marked by [Picards Tools](https://broadinstitute.github.io/picard/) (`rbt call-consensus-from-bam`)
```

Comment:
Maybe rather two subcommands?
`rbt call-consensus-reads [bam|fastq] ...`

Reply:
Good point. That's changed now.